### DISCUSSION TITLE
Change group instead of giving everyone write permisssions?

### DISCUSSION THREAD

**Discussion by Cronvs** [wiiznokes/fan-control#130] [General]
Isn't it safer to only change the group in the necessary udev rules for fan-control?
Could just add the user running it to a group?

**Comment by wiiznokes** [comment]
If you can come up with a solution, yes it will be better. The goup of pwm file will need to be changed from root to the user running. And then, the group permission will need write access.

I made a comment that talk about it here https://github.com/wiiznokes/fan-control/issues/117#issuecomment-2122502881

> We might be able to play with OWNER, GROUP, MODE and TAG+="uaccess" instead of relying on hacky chmod command ([doc](https://www.freedesktop.org/software/systemd/man/latest/udev.html#OWNER,%20GROUP,%20MODE)).

**Reply by Cronvs** [reply]
I can take a shot at it, if that's wanted?

**Reply by wiiznokes** [reply]
Sure. I did some experimenting on the rules branch without success
